examples/data_collect/main.cpp

```diff
@@ -222,7 +220,13 @@ int main(int argc, char *argv[]) {
     n_frames = std::stoi(command_map[{"-n", "--n"}]);
 
     // Parsing mode type
-    mode = std::stoi(command_map[{"-m", "--m"}]);
+    std::string modeName;
+    try {
+        mode = std::stoi(command_map[{"-m", "--m"}]);
+        throw 1;
```

Comment:
Why are you throwing 1?

Reply:
Not throwing 1 anymore. Now it throws the std::string that raised the exception.